subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/walkGraph/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+repositories {
+    jcenter()
+}
+
+val scm by configurations.creating
+
+dependencies {
+    scm("org.eclipse.jgit:org.eclipse.jgit:4.9.2.201712150930-r")
+    scm("commons-codec:commons-codec:1.7")
+    scm("some:unresolved:2.5")
+}
+
+// tag::walk-task[]
+tasks.create<DependencyGraphWalk>("walkDependencyGraph") {
+    dependsOn(configurations.getByName("scm"))
```

Comment:
:x: should be the simpler `dependsOn(scm)`

Reply:
I agree, but I chose to use this for two reasons:
 - the extracted sample in the documentation doesn't show the definition of `scm`, so the reader wouldn't have any idea what `scm` is
 - the groovy example uses `dependsOn configurations.scm`, so using `configurations.getByName("scm")`is closer to groovy, too.

But I realise I could simply use `dependsOn(configurations["scm"])`.
I'll change to that. Tell me if it bothers you.